lib/chai/core/assertions.js

```diff
@@ -679,14 +684,15 @@ module.exports = function (chai, _) {
   /**
    * ### .least(value)
    *
-   * Asserts that the target is greater than or equal to `value`.
+   * Asserts that the number target is greater than or equal to `value`.
```

Comment:
I know this is a merge, but I'll just leave this here as a note:
Shouldn't it be written `Asserts that the **target number** is greater than ...` instead of `Asserts that the **number target** ...